src/chainparams.cpp

```diff
@@ -443,7 +563,7 @@ class CRegTestParams : public CChainParams {
         base58Prefixes[EXT_PUBLIC_KEY] = {0x04, 0x35, 0x87, 0xCF};
         base58Prefixes[EXT_SECRET_KEY] = {0x04, 0x35, 0x83, 0x94};
 
-        bech32_hrp = "bcrt";
+        bech32_hrp = "ccrt";
     }
 
     /**
```

Comment:
This whole file is a mess, needs to be reverted to btc core version and Cloak specific values applied from: https://github.com/CloakProject/codename-phoenix/blob/0.17.cloak/src/chainparams.cpp